.env.template

```diff
@@ -1,7 +1,38 @@
+# AWS Secrets
+# These credentials are baked into the image to be used by the application
 export AWS_ACCESS_KEY_ID="tHiSiSyOuRAcCeSsKeY"
 export AWS_SECRET_ACCESS_KEY="tHiSiSyOuRAcCeSsSeCrEt"
-export DEV_AWS_ACCESS_KEY_ID="tHiSiSyOuRdEvAcCeSsKeY"
-export DEV_AWS_SECRET_ACCESS_KEY="tHiSiSyOuRdEvAcCeSsSeCrEt"
+
+# This is a non-standard name used in this repo;
+# it is tech debt and should be cleaned up to use the standard name
+export AWS_ACCESS_SECRET=$AWS_SECRET_ACCESS_KEY
+
+# These are only used by GitHub Actions, and use secrets of the same names
+# configured on GitHub; they are listed here only for the sake of completeness
+export DEV_AWS_ACCESS_KEY_ID=
+export DEV_AWS_SECRET_ACCESS_KEY=
+export STAGING_AWS_ACCESS_KEY_ID=
+export STAGING_AWS_SECRET_ACCESS_KEY=
+export PROD_AWS_ACCESS_KEY_ID=
+export PROD_AWS_SECRET_ACCESS_KEY=
+
+# These credentials are used by Packer to build the AMI
+# See the item "AWS Github/Terraform build access key" in Bitwarden
+export DEPLOY_AWS_ACCESS_KEY_ID="tHiSiSyOuRAcCeSsKeY"
+export DEPLOY_AWS_SECRET_ACCESS_KEY="tHiSiSyOuRAcCeSsSeCrEt"
+
+# AWS Region
+# For developers, it is easier to clean up if you use a different region than
+# we do for everything else; note that you'll need to switch regions in the
+# console to see your AMI
+# See also https://docs.aws.amazon.com/general/latest/gr/rande.html
 export AWS_REGION=us-east-1
+
+# Permanent environment name; must be one of [local, dev, staging, prod].
+# For developers, keep this as dev.
 export PERM_ENV=dev
```

Comment:
Because this is building an AMI which will run in AWS; I don't know the full list of differences between local and dev, and I haven't tried building a "local" AMI, but I suspect that there is at least one critical difference that would prevent a "local" AMI from working correctly.

...not that we should be manually building images that get pressed into service, but I'm fairly certain that's at least possible with dev, and I'm fairly skeptical with local.

Reply:
Gotcha, thanks!